graspologic/embed/ase.py

```diff
@@ -158,4 +160,63 @@ def fit(self, graph, y=None):
 
         self.n_features_in_ = len(A)
         self._reduce_dim(A)
+
+        # for out-of-sample
+        inv_eigs = np.diag(1 / self.singular_values_)
+        self.pinv_left_ = self.latent_left_ @ inv_eigs
+        if self.latent_right_ is not None:
+            self.pinv_right_ = self.latent_right_ @ inv_eigs
```

Comment:
putting this in `fit` adds extra attributes that sometimes won't be used, but we don't want to compute this for every prediction if e.g. predict gets run in a loop

Reply:
should document or make private (don't remember seeing this but maybe I missed?)

Reply:
by make private, do you mean making the attribute something like `_pinv_left` instead of `pinv_left_`?

Reply:
see 5f36240